addons/website_sale/views/templates.xml

```diff
@@ -779,18 +779,24 @@
                     <t t-if="category.parent_id" t-set="backUrl" t-value="keep('/shop/category/' + slug(category.parent_id), category=0)"/>
                     <t t-else="" t-set="backUrl" t-value="'/shop'"/>
 
-                    <li t-foreach="entries" t-as="c"
-                        t-attf-class="d-flex {{'pe-3' if not c_last else ''}}"
-                        t-att-data-link-href="keep('/shop/category/' + slug(c), category=0)">
-                        <input type="radio" t-attf-name="wsale_categories_top_radios_{{parentCategoryId}}" class="btn-check pe-none" t-att-id="c.id" t-att-value="c.id" t-att-checked="'true' if c.id == category.id else None"/>
-
-                        <div t-attf-class=" btn btn-{{navClass}} d-flex align-items-center {{'ps-2 pe-3' if c.image_128 else 'px-4'}} fs-6 fw-normal {{ 'border-primary' if c.id == category.id else '' }}" t-att-for="c.id">
-                            <div t-if="c.image_128"
-                                 t-attf-style="background-image:url('data:image/png;base64,#{c.image_128}')"
-                                 class="o_image_40_cover oe_img_bg o_bg_img_center rounded-3 me-3"
-                                 t-att-alt="c.name "/>
-                            <span t-field="c.name"/>
-                        </div>
+                    <li t-foreach="entries" t-as="c" t-attf-class="d-flex {{'pe-3' if not c_last else ''}}">
+                        <a t-att-href="keep('/shop/category/' + slug(c), category=0)" class="text-decoration-none">
```

Comment:
In which case does `text-decoration-none` do something here?

Reply:
Since it is an `<a>` tag, the text will be underlined when we hover over that element.
![image](https://github.com/user-attachments/assets/63d8b15a-7c2e-42a5-8caa-e056fb2703b3)